src/lib/components/Card.svelte

```diff
@@ -10,13 +10,14 @@
 
 <div
 	class="flex items-center gap-x-6 shadow-xl p-4 my-2 rounded-lg hover:scale-105 hover:bg-[#0f172ac6] hover:text-white duration-300"
+	
```

Comment:
If you have the prettier extension press **alt shift F** to make the code cleaner.